src/app/_components/Feedback/Feedback.js

```diff
@@ -59,15 +59,15 @@ export function Feedback({ isFeedbackOpen, onClose }) {
       onClose();
     }, 4000);
 
-    // Existing lint rules do not allow this usage, but this code is required.
+    // that because of first return void, and second return clearTimeout
```

Comment:
You can just return `undefined` from the first return, to avoid disabling eslint.

Reply:
done